### PR TITLE
Update GitHub URLs to point to new org

### PR DESCRIPTION
**Pull request by alexdewar**
The org name was changed from SGIModel to EnergySystemsModellingLab at some point, but URLs in the code haven't been updated. While old URLs will still redirect to the right place, it's tidier if we just use the new ones. Fix this.

@ahawkes I've noticed that the author is still listed for the project as Sustainable Gas Institute with the email sgi@imperial.ac.uk. Do you have a new name/email address we can use? This information will show up on PyPI, so it is a branding thing.

For other projects we do sometimes also add the RSE's email address just to have it as a point of contact, but I dunno if we want to do that.

### CODE REVIEW

**Comment by ahawkes** [comment]
Thanks
Can we change author to Hawkes Research Group @ Chemical Engineering, Imperial College London. And my email. Also fine to add RSE's email!
A


________________________________
From: Alex Dewar ***@***.***>
Sent: Thursday, June 20, 2024 9:03 AM
To: EnergySystemsModellingLab/MUSE_OS ***@***.***>
Cc: Hawkes, Adam D ***@***.***>; Mention ***@***.***>
Subject: [EnergySystemsModellingLab/MUSE_OS] Update GitHub URLs to point to new org (PR #353)

This email from ***@***.*** originates from outside Imperial. Do not click on links and attachments unless you recognise the sender. If you trust the sender, add them to your safe senders list<https://spam.ic.ac.uk/SpamConsole/Senders.aspx> to disable email stamping for this address.



The org name was changed from SGIModel to EnergySystemsModellingLab at some point, but URLs in the code haven't been updated. While old URLs will still redirect to the right place, it's tidier if we just use the new ones. Fix this.

@ahawkes<https://github.com/ahawkes> I've noticed that the author is still listed for the project as Sustainable Gas Institute with the email ***@***.******@***.***>. Do you have a new name/email address we can use? This information will show up on PyPI, so it is a branding thing.

For other projects we do sometimes also add the RSE's email address just to have it as a point of contact, but I dunno if we want to do that.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353

Commit Summary

  *   91ad1c8<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/commits/91ad1c8cb1db715488e944633bd320ebef779e1a> Update GitHub URLs to point to new org

File Changes

(15 files<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files>)

  *   M .github/PULL_REQUEST_TEMPLATE.md<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35> (2)
  *   M CHANGELOG.md<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed> (132)
  *   M README.md<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (2)
  *   M docs/getting-help.rst<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-2a40b960ebc5972ced282c20a45d38b841dc841fb9dbaccd291f60225f06d999> (8)
  *   M docs/installation/developers.rst<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-439aa93779072ee0bac1c0b0f7a2cc75bff0bb364a72f078653776fda5ad3d8a> (2)
  *   M docs/installation/standalone-muse.rst<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-df7a5fe4a7ef24ce3d797db2a2aaa0bf6d51cf5fc4546806fccc11540dc9ea49> (2)
  *   M docs/running-muse-example.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-010151300cb2afe3b94b1eb25e4052bdead24e303d3ff0a1be56944aa2d47961> (2)
  *   M docs/user-guide/add-agent.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-d152097fa21173f08b6ba007794037183170f669f505da76a40f0805c770c280> (2)
  *   M docs/user-guide/add-gdp-correlation-demand.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-b9d9723d1e5289c401828787bec554a2939153719baadcf62608f99e268aea73> (6)
  *   M docs/user-guide/add-region.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-c9f862b407c25752177183f1c04cfd5714e10405524bef0255017303f698f321> (6)
  *   M docs/user-guide/add-solar.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-4aab21639639fbe387502c09eb1f1246608c00055efa7af7e3d0af959bfff870> (8)
  *   M docs/user-guide/additional-service-demand.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-22662347e0041bb017d8b1f2289553a1d0f1b400188722dcc89ba07ef07e7140> (6)
  *   M docs/user-guide/index.rst<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-de7873f447f4594e169e2776cc7e9beef03aaf8f96ca89b1dca8df1eeacbdb5e> (14)
  *   M docs/user-guide/modify-timing-data.ipynb<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-56b396cfe56cbe77293385e41bf9ee305b6d7e16481134176273b80524a753f4> (4)
  *   M src/muse_gui/__main__.py<https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353/files#diff-b454f5cd26e02e50de9a18e1811c77b0d8fa7801dee08e2e5327cb730789d82d> (2)

Patch Links:

  *   https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353.patch
  *   https://github.com/EnergySystemsModellingLab/MUSE_OS/pull/353.diff

**Comment by alexdewar** [comment]
@ahawkes Okey doke. Done

**Review by dalonsoa — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Fixes # (issue)
 ## Type of change
 
 Please add a line in the relevant section of
-[CHANGELOG.md](https://github.com/SGIModel/StarMuse/blob/development/CHANGELOG.md) to
+[CHANGELOG.md](https://github.com/EnergySystemsModellingLab/StarMuse/blob/development/CHANGELOG.md) to
```
Comment (tsmbland) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
[CHANGELOG.md](https://github.com/EnergySystemsModellingLab/MUSE_OS/blob/development/CHANGELOG.md) to
```

A few other references to `StarMuse` which may also need to be changed. (Also I'm not sure this comment about the changelog is still relevant, but I guess that's a separate issue)

Comment (tsmbland) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Actually I think all the other references to `StarMuse` are correct, but this one needs to be changed

Comment (alexdewar) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Good spot! Thank you for reading through this more carefully than I did :laughing:

Comment (alexdewar) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Not that we actually use the changelog anymore, but it might as well be correct!

**Pull request merged**